src/commands/create/run.js

```diff
@@ -1,68 +1,142 @@
 'use strict';
 
 const fs = require('fs');
+const got = require('got');
+const ora = require('ora');
 const path = require('path');
-const nrd = require('nrd');
+const unzip = require('unzip');
+const exec = require('child_process').exec;
 
-const SCAFFOLD_PROJECT = 'marko-starter-demo';
+const DEFAULT_REPO = 'demo';
+const MARKO_PREFIX = 'marko-';
+const MARKO_STARTER_PREFIX = 'marko-starter-';
+const MARKO_SAMPLES_ORG = 'marko-js-samples';
+const GITHUB_URL = 'https://github.com/';
+const ARCHIVE_PATH = '/archive/master.zip';
+const MASTER_SUFFIX = '-master';
 
+module.exports = function run(options, devTools) {
+    console.log('');
```

Comment:
The blank line here actually is intentional.  Whether or not it's necessary could be up for debate though.